experiment.csv

```diff
@@ -1,22 +1,22 @@
-_authors,Denis Pelli,Denis Pelli,Denis Pelli,Denis Pelli,Denis Pelli,Denis Pelli
-block,1,1,1,1,2,2
-calibrateTrackDistanceBool,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE
-conditionName,printLetters,printString,scriptLetters,scriptString,arabicLetters,arabicString
-conditionTrials,3,3,3,3,3,3
-showCounterBool,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE
-spacingDirection,radial,radial,radial,radial,radial,radial
-spacingOverSizeRatio,1.4,1.4,1.4,1.4,1.4,1.4
-spacingRelationToSize,ratio,typographic,ratio,typographic,ratio,typographic
-targetAlphabet,DHKNORSVZ,DHKNORSVZ,abcdefghijklmnopqrstuvwxyz,abcdefghijklmnopqrstuvwxyz,ابتثجحخدذرزسشصضطظعغفقكلمنهوي,ابتثجحخدذرزسشصضطظعغفقكلمنهوي
-targetDurationSec,1,1,1,1,1,1
-targetEccentricityXDeg,-10,10,-10,10,-10,10
-targetEccentricityYDeg,0,0,0,0,0,0
-targetFont,Sloan.woff2,Sloan.woff2,Douglass Pen.woff2,Douglass Pen.woff2,IBMPlexSansArabic.woff2,IBMPlexSansArabic.woff2
-targetMinimumPix,8,8,8,8,8,8
-targetTask,identify,identify,identify,identify,identify,identify
-thresholdBeta,2.3,2.3,2.3,2.3,2.3,2.3
-thresholdDelta,0.01,0.01,0.01,0.01,0.01,0.01
-thresholdGuess,2,2,2,2,2,2
-thresholdGuessLogSd,3,3,3,3,3,3
-thresholdParameter,spacing,spacing,spacing,spacing,spacing,spacing
-viewingDistanceDesiredCm,40,40,40,40,40,40
+_authors,Denis Pelli,Denis Pelli,Denis Pelli,Denis Pelli,Denis Pelli,Denis Pelli
```

Comment:
The file looks unchanged, yet every line is changed, can you try to copy from the original raw and see if it still shows modified?